src/test/javascript/portal/details/InfoPanelSpec.js

```diff
@@ -9,7 +9,7 @@ describe("Portal.details.InfoPanel", function() {
 
     var panel;
     var abstractTitle = OpenLayers.i18n('abstractTitle');
-    var webpageLinksTitle = OpenLayers.i18n('webpageLinksTitle');
+    var supplementaryLinksTitle = OpenLayers.i18n('supplementaryLinksTitle');
```

Comment:
This is a better name, I couldn't think of one when I came up with the very creative `webpage`.